### PR TITLE
feat(cni): watch service change and update ovs flow

### PR DESCRIPTION
**Pull request by qingwusunny**

### CODE REVIEW

File: pkg/agent/datapath/natBridge.go [hunk 1]
```diff
@@ -216,13 +273,233 @@ func (n *NatBridge) DelLBIP(svcID, ip string) error {
...
+		if err != nil {
+			log.Errorf("Failed to add session affinity flow for service %s ip %s, port %s, err: %s", svcID, ips[i], portName, err)
 			return err
 		}
-		svcOvsCache.DelLBFlow(ip, lbFlows[i].PortName)
+		svcOvsCache.SetSessionAffinityFlow(ips[i], portName, sessionFlow)
 	}
-	// todo process sessionAffinity
+
+	log.Infof("Dp success AddLBPort for service %s port %s", svcID, portName)
```
Comment (KeyongSun) [inline pkg/agent/datapath/natBridge.go:364]: generally not export the func name in the log.

File: pkg/agent/datapath/natBridge.go [hunk 2]
```diff
@@ -216,13 +273,233 @@ func (n *NatBridge) DelLBIP(svcID, ip string) error {
...
+		log.Errorf("Failed to delete old session affinity flow for service %s port %s, err: %s", svcID, port.Name, err)
+		return err
+	}
+
+	// add new flows
+	if err := n.AddLBPort(svcID, port, ips, sessionAffinityTimeout); err != nil {
+		log.Errorf("Failed to add new flow for service %s update port %+v, err: %s", svcID, *port, err)
+		return err
+	}
+	log.Infof("Dp success UpdateLBPort for service %s port %s", svcID, port.Name)
```
Comment (KeyongSun) [inline pkg/agent/datapath/natBridge.go:393]: Ditto

File: pkg/agent/datapath/natBridge.go (hunk 1 above)
Comment (qingwusunny) [reply pkg/agent/datapath/natBridge.go]: done

File: pkg/agent/datapath/natBridge.go (hunk 2 above)
Comment (qingwusunny) [reply pkg/agent/datapath/natBridge.go]: done

**Review by KeyongSun — APPROVED** [review]

**Review by echkenluo — APPROVED** [review]

**Pull request merged**